custom/icds_reports/migrations/0016_aggawcmonthly_aggccsrecordmonthly_aggchildhealthmonthly_aggdailyusageview_aggthrmonthly_awclocation_.py

```diff
@@ -14,7 +33,7 @@ class Migration(migrations.Migration):
         ('icds_reports', '0015_add_aggregation_level'),
     ]
 
-    operations = [
+    operations = get_view_migrations() + [
```

Comment:
@emord FYI, I've moved `create_views` operations to here.

Reply:
Actually had to rollback. I tried to make creating views via migration, but the migration dependency made it difficult.